lib/facility.rb

```diff
@@ -1,14 +1,65 @@
 class Facility
-  attr_reader :name, :address, :phone, :services
+  attr_reader :name, :address, :phone, :services, :registered_vehicles, :collected_fees
 
-  def initialize(name, address, phone)
-    @name = name
-    @address = address
-    @phone = phone
+  def initialize(args)
+    @name = args[:name]
+    @address = args[:address]
+    @phone = args[:phone]
     @services = []
+    @registered_vehicles = []
+    @collected_fees = 0
   end
 
-  def add_services(service)
+  def add_service(service)
     @services << service
   end
+  
+  def register_vehicle(vehicle)
+    if @services.include?('Vehicle Registration')
+    @registered_vehicles << vehicle
+    vehicle.registration_date == Date.today
+     if vehicle.antique? == true
+      @collected_fees += 25
+      vehicle.plate_type = :antique
+     elsif vehicle.electric_vehicle? == true
+      @collected_fees += 200
+      vehicle.plate_type = :ev
+     else
+      @collected_fees += 100
+      vehicle.plate_type = :regular
+     end
+    else
+    end
+  end
```

Comment:
Overall, this method looks good, it's doing all the things we need this method to do. A few notes:
1. The indentation on this method is all off. Line 21 should not be spaced in. Also, the indentation appears to sometimes be a tab and sometimes a space, we want to be consitent and always use a tab.
2. We could refactor this method by taking lines 21-30 and putting those in their own helper method, then just calling the helper method from line 21. We want to start working on having each method doing only one thing.